src/test/resources/pacts/publicapi-ledger-search-payment-by-first-digits-card-number.json

```diff
@@ -7,7 +7,7 @@
   },
   "interactions": [
     {
-      "description": "search payments",
+      "description": "search payment with first digits card number",
```

Comment:
made a change to this description to differentiate the tests. i was getting confused when running the ledger provider test locally.